pydm/utilities/stylesheet.py

```diff
@@ -0,0 +1,128 @@
+# Utility to handle importing the global stylesheet for PyDM widgets
+
+from ..PyQt.QtGui import QApplication
+import os
+from functools import partial
+
+import logging
+logger = logging.getLogger(__name__)
+from . import is_pydm_app
+
+
+# Fallback global stylesheet if there is no global stylesheet provided via env variable or command line parameter
+# TODO: Expand this global stylesheet
+GLOBAL_STYLESHEET = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../default_stylesheet.qss')
```

Comment:
This will probably break with windows... We should use the `os.sep` instead of the `/` and wrap it all with: `os.path.realpath` so we can have a path that is direct to the file and not relative.